### ISSUE TITLE
second order triangular element

### ISSUE THREAD

**Issue by heavenlyhosts128** [pyvista/pyvista-support#225]
## Description
Hi! 

I load a vtk file using pyvista. My file has second-order triangular elements (CELL_TYPES = 22). When I plot it and use show_edges=True I get the following:

![pyvtk](https://user-images.githubusercontent.com/48632221/89790005-9f39f200-db21-11ea-837c-59b30671d5f4.jpg)

But this is not ok. If I plot it in Paraview and display it with surface with edges I get the correct cells.

![paraview](https://user-images.githubusercontent.com/48632221/89790315-0a83c400-db22-11ea-9f7f-2268bd2b4f46.jpg)

Is there any way that I get the correct cells including the edges using pyvista?

/H

## Example Data
Here is my code:

    import pyvista as pv

    # read the data
    grid = pv.read('step2_wWater_sobol.vtk')
    # plot the data with an automatically created Plotter
    grid.plot(scalars='ST_-sigyy_geo_lambda_star', show_scalar_bar=True, 
    cpos="xy", show_edges=True)

**Comment by banesullivan** [comment]
Can you share the data file?

**Comment by banesullivan** [comment]
Actually, we have a quadratic example in `examples.download_tri_quadratic_hexahedron()`

```py
from pyvista import examples
mesh = examples.download_tri_quadratic_hexahedron()
```

I'm currently not aware of an immediately obvious solution. ParaView seems to do some magic with a surface extraction filter.

See https://discourse.vtk.org/t/hide-internal-edges-of-quadratic-cells/2419

**Comment by heavenlyhosts128** [comment]
Yes sure, here I attach the data file. Note that I change the .vtk to .txt in order to upload it.

[sample.txt](https://github.com/pyvista/pyvista-support/files/5056345/sample.txt)

I will also check out the example you've provided to see if I can work it out or not.

**Comment by banesullivan** [comment]
I'm going to move this to the PyVista repo as a feature request
